github/lib/configure_repo.rb

```diff
@@ -1,9 +1,10 @@
 class ConfigureRepo
   attr_reader :repo, :client
 
-  def initialize(repo, client)
+  def initialize(repo, client, overrides = nil)
     @repo = repo
     @client = client
+    @overrides = overrides || {}
   end
```

Comment:
Is it just a matter of preference or is there any benefit in doing this instead of
```
def initialize(repo, client, overrides = {})
  # ...
  @overrides = overrides
end
```
?

Reply:
If an override doesn't exist in the YAML file, `overrides[repo]` will be `nil`. So we need to have `@overrides = overrides || {}`.

I decided to leave the default as `nil` because I wanted to retain the close-as-possible behaviour from before, but I'm not too fused here on what the default is.

Reply:
Right, even if we had `def initialize(repo, client, overrides = {})`, when we call `ConfigureRepo.new(repo, client, repo_overrides[repo])` with a repo not present in the yml file, we would be calling `ConfigureRepo.new(repo, client, nil)` and so we would still need `@overrides = overrides || {}`.